ssz/sedes/base.py

```diff
@@ -104,77 +83,126 @@ def __init__(self, size: int):
     #
     # Size
     #
-    @property
-    def is_static_sized(self):
-        return True
+    is_fixed_sized = True
```

Comment:
I don't think it's a concern.  I know historically I may have advocated for this safety, and I'm not opposed to adding it, but since there's nowhere in the codebase that assigns to this property I don't see it being something that's likely to go wrong.  Happy to change it back if someone prefers it to be done with `@property`

Reply:
No, to be honest I don't care too much either way.